grails-app/services/au/org/emii/gogoduck/job/JobExecutorService.groovy

```diff
@@ -61,6 +62,10 @@ class JobExecutorService {
         notificationService.sendJobFailureNotification(job)
     }
 
+    def setJobStatusLink(job) {
+        job.statusLink = notificationService.getJobUrl(job)
```

Comment:
It doesn't seem quite right to me to be "getting" something from the `notificationService`...  It'd be unnecessary if the job url was tacked on on the fly...